File: APIflow/js/tutorials/simple.js
var email = ""
var password = ""
var apiKey = ""

var sessionKey = auth(email, password, apiKey)
var processTypes = listProcessTypes(sessionKey)
for(var i = 0; i < processTypes.length; i++){
	if(processTypes[i].name == "simple"){
		var procAck = startProcess(sessionKey, "simple")
		var process = getProcess(sessionKey, procAck.id)

		for(var t = 0; t < process.tasks.length; t++){
			var res = completeTask(sessionKey, t.ID);
			console.log(res);
		}

	}
}

File: APIflow/ruby/tutorials/simple.rb
require_relative '../APIFlow.rb'

email, password, apiKey = ARGV[0], ARGV[1], ARGV[2]

flow = APIFlow.new(email, password, apiKey)

sessionKey = flow.sessionKey()

processtypes = flow.listProcesstypes(sessionKey)

for p in processtypes
	if p["name"] == "simple"
		procAck = flow.startProcess(sessionKey, p["name"])
		process = flow.getProcess(sessionKey, procAck["id"])
		for t in process["TASKS"]
			res = flow.completeTask(sessionKey, t["ID"])
			print(res)
		end
	end		
end